### ISSUE TITLE
Code can't run. 

### ISSUE THREAD

**Issue by dmhuang7** [jiangxinke/ProtoMix#1]
def forward_train(self,**kwargs):
        patient_embed, mask, patient_emb_all_step = self.model(**kwargs)
        patient_embed = self.dropout(patient_embed)
        patient_y_true = self.model.prepare_labels(kwargs[self.model.label_key], self.model.label_tokenizer)
        
        distances = torch.cdist(patient_embed, self.prototype_embed.clone().detach())
        labels = torch.argmin(distances, dim=1)
        prototype_labels_one_hot = one_hot(labels, self.num_prototypes).to(self.device)

        mixed_embed, labels_reweighted = self.mixup_process(patient_embed, prototype_labels_one_hot,
                                                       adaptive_alpha=True, mixup_alpha=self.mixup_alpha)
        proto_logits = self.proto_fc(mixed_embed)
        loss_protolabel = F.cross_entropy(proto_logits, labels_reweighted)       ####### Is there a problem with the code here
      
       ####### Is there a problem with the code here 
        sample_indices = (self.prototype_embed.clone().detach(), self.positive_frequency, self.negative_frequency, patient_embed,
                                          kwargs[self.model.label_key], self.sample_ratio)
        patient_embed_sample = patient_embed[sample_indices]
        patient_y_true_sample = patient_y_true[sample_indices]

       ####### Is there a problem with the code here， cls_logits have the label space, but labels_reweighted is the prototype space
        mixed_embed, labels_reweighted = self.mixup_process(patient_embed_sample, patient_y_true_sample,
                                                       adaptive_alpha=True, mixup_alpha=self.mixup_alpha)
        cls_logits = self.class_linear(mixed_embed)
        loss_protocls = F.binary_cross_entropy_with_logits(cls_logits,labels_reweighted)

        patient_embed_final = patient_embed
        origin_cls_logits = self.class_linear(patient_embed_final)
        patient_y_prob = self.model.prepare_y_prob(origin_cls_logits)
        loss_origincls = F.binary_cross_entropy_with_logits(origin_cls_logits, patient_y_true)

        return loss_protolabel, loss_protocls, loss_origincls, patient_y_true, patient_y_prob, patient_embed

**Comment by jiangxinke** [comment]
请问您的报错是什么？
